securesign/secure-sign-operator
# feat: add service monitor to ctlog

This pr adds service monitors to ctlog.

# Testing

1. Deploy operator
2. Create CRD with this in it:
```
  ctlog:
    monitoring:
      enabled: true
```
3. Targets will come up, and you can see metrics in Observe ->  Metrics
